### ISSUE TITLE
App Store still flagging privacy issues

### ISSUE THREAD

**Issue by BenTincher** [permutive-engineering/permutive-ios-spm#13]
We've updated our app to use v2.1.1 in order to get the new privacy requirements but Apple is still flagging the Permutive API.  Here is the response received upon submission:

<img width="727" alt="image" src="https://github.com/permutive-engineering/permutive-ios-spm/assets/138822929/20d70e0f-a873-44ab-9a89-787fe7085b1d">

**Comment by TevJ** [comment]
Hi @BenTincher,

Thanks for raising. That's strange as the key was added through Xcode tooling. I've updated the key manually now to include the correct prefix in [Release v2.1.2](https://github.com/permutive-engineering/permutive-ios-spm/releases/tag/v2.1.2).

Apologies for the issue.

**Comment by BenTincher** [comment]
Agreed, I was very confused as well.  Thank you for the quick fix.  We'll try this in the our next release.
